x-pack/solutions/search/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_service_provider/translations.ts

```diff
@@ -13,3 +13,17 @@ export const MIT_LICENSE = i18n.translate(
     defaultMessage: 'License: MIT',
   }
 );
+
+export const ELASTIC_DESCRIPTION = i18n.translate(
```

Comment:
super duper nit: I think it's clearer to rename the constants here to be like TOKEN_BASED vs RESOURCE_BASED hear instead of Elastic and Elasticsearch 🤔